x/oracle/keeper/upgrade.go

```diff
@@ -67,3 +73,20 @@ func (k Keeper) UpgradeOracle(ctx sdk.Context, msg *types.MsgUpgradeOracle) erro
 	)
 	return nil
 }
+
+func (k Keeper) ApplyUpgrade(ctx sdk.Context, info *types.OracleUpgradeInfo) error {
```

Comment:
Why don't we add a log when oracle upgrade is applied?

Reply:
Thank you!
I added a log!(https://github.com/medibloc/panacea-core/pull/472/commits/b88c8912f70630aa1fd1ea9deb72198d6893870f)